<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObsidianInstaller::MarkdownRenderer</class>
 <widget class="QWidget" name="ObsidianInstaller::MarkdownRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkdownRenderer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="textEdit">
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
